UniAtHome/UniAtHome.DAL/Migrations/UniAtHomeDbContextModelSnapshot.cs

```diff
@@ -19,6 +19,137 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                 .HasAnnotation("Relational:MaxIdentifierLength", 128)
                 .HasAnnotation("SqlServer:ValueGenerationStrategy", SqlServerValueGenerationStrategy.IdentityColumn);
 
+            modelBuilder.Entity("Microsoft.AspNetCore.Identity.IdentityRole", b =>
```

Comment:
Please, don't modify autogenerated tables

Reply:
It has automagically updated after Migration-Add. I don't even know what it is :P